src/ExpenseTracker.Core/Entities/User.cs

```diff
@@ -33,7 +34,8 @@ public User(string firstName, string lastName, string username, string password)
         public virtual bool HasWorkspace => Workspaces.Any();
 
         public virtual bool HasDefaultWorkspace =>
-            Workspaces.Count(a => a.WorkspaceType == Workspace.TypeDefaultWorkspace) == 1;
+            Workspaces.Count(a =>
```

Comment:
@BirajMainali Using `Any` might be a better idea instead of count and number check

Reply:
![ArreyKehnaKyaChahteHoWhatIsMachineSceneEngineeringGIF](https://user-images.githubusercontent.com/71131016/140650200-eb4a67a9-b50e-4f32-95e8-0d1f32d2c240.gif)